get-qa-ssh-keys.sh

```diff
@@ -9,7 +9,6 @@ umask 077
 aws s3 sync s3://${SSH_KEYS_BUCKET} ${SSH_KEYS_DIR} --exact-timestamps --exclude '*' \
   --include scylla-qa-ec2 \
   --include scylla-test \
-  --include support \
```

Comment:
This support key isn't gonna be used at all ?

Let's check if we can remove it completely (and stop downloading it also on SCT release branches)

Reply:
I think we need to keep it in keystore, but stop to download in release branches.

Reply:
So let's put this in a separate commit so we can backport

Reply:
It's already done, btw :)